x-pack/filebeat/module/sonicwall/firewall/config/liblogparser.js

```diff
@@ -1007,8 +1007,8 @@ var ecs_mappings = {
     "child_process": {to:[{field: "process.name", setter: fld_prio, prio: 1}]},
     "city.dst": {to:[{field: "destination.geo.city_name", setter: fld_set}]},
     "city.src": {to:[{field: "source.geo.city_name", setter: fld_set}]},
-    "daddr": {convert: to_ip, to:[{field: "destination.ip", setter: fld_append},{field: "related.ip", setter: fld_append}]},
-    "daddr_v6": {convert: to_ip, to:[{field: "destination.ip", setter: fld_append},{field: "related.ip", setter: fld_append}]},
+    "daddr": {convert: to_ip, to:[{field: "destination.ip", setter: fld_set},{field: "related.ip", setter: fld_append}]},
```

Comment:
Why set and not append? Normally related.ip is an array. This seems unrelated to adding network direction.

Reply:
i don't recall removing `"daddr_v6": {convert: to_ip, to:[{field: "destination.ip", setter: fld_append},{field: "related.ip", setter: fld_append}]},`.  That is probably an accident.  As for the change from `{field: "destination.ip", setter: fld_append}` to `{field: "destination.ip", setter: fld_set}`, is because the `network_direction` processor requires a string field not an array.

Reply:
The append for `related.ip` is unchanged.

Reply:
I am reverting the changes for Snort and Sonicwall.

Reply:
removed

Reply:
Thanks for pointing out at source.ip and destination.ip should be scalars. I think we should make that change just not in this PR. Would you please open an issue for this? It does look like you reverted most of the snort/sonicwall changes, but a small part of snort is still changed.